packages/legend-query/src/components/QueryBuilderFunctionPanel.tsx

```diff
@@ -0,0 +1,653 @@
+  BlankPanelContent,
+  MenuContentItem,
+  DropdownMenu,
+  MenuContent,
+  CheckIcon,
+  MenuContentItemIcon,
+  MenuContentItemLabel,
+  MoreVerticalIcon,
+  clsx,
+} from '@finos/legend-art';
+import {
+  type QueryBuilderPackageElementTreeNodeData,
+  type QueryBuilderFunctionDragSource,
+  QUERY_BUILDER_FUNCTION_TREE_DND_TYPE,
+  generateQueryBuilderPackableElementTreeNodeData,
+  getTreeChildNodes,
+  generateFunctionSignature,
+} from '../stores/QueryFunctionsState';
+import { useDrag, useDragLayer } from 'react-dnd';
+import {
+  ConcreteFunctionDefinition,
+  Package,
+  ROOT_PACKAGE_NAME,
+} from '@finos/legend-graph';
+import { getEmptyImage } from 'react-dnd-html5-backend';
+import { useEffect, useState } from 'react';
+import type { QueryBuilderState } from '../stores/QueryBuilderState';
+import { getMultiplicityDescription } from './shared/QueryBuilderUtils';
+
+const QueryBuilderFunctionInfoTooltip: React.FC<{
+  element: ConcreteFunctionDefinition;
+  children: React.ReactElement;
+  placement?: TooltipPlacement | undefined;
+}> = (props) => {
+  const { element, children, placement } = props;
+  return (
+    <Tooltip
+      arrow={true}
+      {...(placement !== undefined ? { placement } : {})}
+      classes={{
+        tooltip: 'query-builder__tooltip',
+        arrow: 'query-builder__tooltip__arrow',
+        tooltipPlacementRight: 'query-builder__tooltip--right',
+      }}
+      TransitionProps={{
+        timeout: 0,
+      }}
+      title={
+        <div className="query-builder__tooltip__content">
+          <div className="query-builder__tooltip__item">
+            <div className="query-builder__tooltip__item__label">
+              Function Name
+            </div>
+            <div className="query-builder__tooltip__item__value">
+              {element.path}
+            </div>
+          </div>
+          <div className="query-builder__tooltip__item">
+            <div className="query-builder__tooltip__item__label">
+              Parameters
+            </div>
+            <div className="query-builder__tooltip__item__value">
+              {element.parameters
+                .map(
+                  (t) =>
+                    `${t.name}: ${
+                      t.type.value.name
+                    }${getMultiplicityDescription(t.multiplicity)}`,
+                )
+                .join('; ')}
+            </div>
+          </div>
+          <div className="query-builder__tooltip__item">
+            <div className="query-builder__tooltip__item__label">
+              Documentation
+            </div>
+            <div className="query-builder__tooltip__item__value">
+              {element.taggedValues
+                .filter((t) => t.tag.value.owner.name === 'doc')
+                .map((t) => t.value)
+                .join('; ')}
+            </div>
+          </div>
+          <div className="query-builder__tooltip__item">
+            <div className="query-builder__tooltip__item__label">
```

Comment:
Showing `tagged values` might be a little too cluttered. What do you think @MauricioUyaguari?

Reply:
Could be. What if we just show if it has documentation. `doc.doc`?